ui/components/app/nfts-tab/nfts-tab.js

```diff
@@ -60,89 +55,53 @@ export default function NftsTab({ onAddNFT }) {
       )}
       <Box
-        marginBottom={4}
-        justifyContent={JustifyContent.center}
-        flexDirection={FLEX_DIRECTION.COLUMN}
+        className="nfts-tab__buttons"
+        display={Display.Flex}
+        flexDirection={FlexDirection.Column}
+        alignItems={AlignItems.flexStart}
+        margin={4}
+        gap={2}
+        marginBottom={2}
       >
-        <Typography
-          color={TextColor.textMuted}
-          variant={TypographyVariant.H5}
-          align={TextAlign.Center}
-        >
-          {t('missingNFT')}
-        </Typography>
-        <Box
-          alignItems={AlignItems.center}
-          justifyContent={JustifyContent.center}
+        <ButtonLink
+          size={Size.MD}
+          data-testid="import-nft-button"
+          startIconName={IconName.Add}
+          onClick={onAddNFT}
         >
-          {!isMainnet && Object.keys(collections).length < 1 ? null : (
-            <>
-              <Box
-                className="nfts-tab__link"
-                justifyContent={JustifyContent.flexEnd}
-              >
-                {isMainnet && !useNftDetection ? (
-                  <Button type="link" onClick={onEnableAutoDetect}>
-                    {t('enableAutoDetect')}
-                  </Button>
-                ) : (
-                  <Button type="link" onClick={onRefresh}>
-                    {t('refreshList')}
-                  </Button>
-                )}
-              </Box>
-              <Typography
-                color={TextColor.textMuted}
-                variant={TypographyVariant.H6}
-                align={TextAlign.Center}
-              >
-                {t('or')}
-              </Typography>
-            </>
-          )}
-          <Box
-            justifyContent={JustifyContent.flexStart}
-            className="nfts-tab__link"
-          >
-            <Button type="link" onClick={onAddNFT}>
-              {t('importNFTs')}
-            </Button>
-          </Box>
-        </Box>
+          {t('importNFT')}
+        </ButtonLink>
+        {!isMainnet && Object.keys(collections).length < 1 ? null : (
+          <>
+            <Box
+              className="nfts-tab__link"
+              justifyContent={JustifyContent.flexEnd}
+            >
+              {isMainnet && !useNftDetection ? (
+                <ButtonLink
+                  size={Size.MD}
+                  startIconName={IconName.Setting}
+                  data-testid="refresh-list-button"
+                  onClick={onEnableAutoDetect}
+                >
+                  {t('enableAutoDetect')}
+                </ButtonLink>
+              ) : (
+                <ButtonLink
+                  size={Size.MD}
+                  startIconName={IconName.Refresh}
+                  data-testid="refresh-list-button"
+                  onClick={onRefresh}
+                >
+                  {t('refreshList')}
+                </ButtonLink>
```

Comment:
I think we can incorporate this as well!

https://github.com/MetaMask/metamask-extension/pull/19743

Reply:
I think this already exists here for NFTs

https://github.com/MetaMask/metamask-extension/assets/39872794/105032f0-f274-4aa3-8bf2-b08a6b16db1c